src/components/Dialog-Window/Notes/DialogNote.tsx

```diff
@@ -64,6 +71,16 @@ export const Note: React.FC<NoteProps> = ({ maxLength, id }) => {
           defaultValue={note}
         />
         <p className={styles.dynamicSavingText}>{dynamicSavingText}</p>
+        <div className={styles.markAsCompletedCheckbox}>
+          <label htmlFor="note">
```

Comment:
This is not the label for `note`, but for the element inside. The `htmlFor` can be removed.

```suggestion
          <label>
```

Reply:
Updated label with proper control, eslint complains if there's no association for label so I've connected it with the input (checkbox).